xds/internal/xdsclient/transport/ads/ads_stream.go

```diff
@@ -385,18 +378,10 @@ func (s *StreamImpl) sendExisting(stream transport.StreamingCall) error {
 			continue
 		}
 
-		names := resourceNames(state.subscribedResources)
-		if s.logger.V(2) {
-			s.logger.Infof("Re-requesting resources %v of type %q, as the stream has been recreated", names, typ.TypeURL())
-		}
```

Comment:
Should this be put on line 381 now?

Reply:
I decided to get rid of this log message since the resources are anyway logged from `sendMessageLocked`.